_P177_CM1107.ino

```diff
@@ -0,0 +1,235 @@
+//#######################################################################################################
+//######################## Plugin 177 CM1107 I2C CO2 Sensor ############################################
+//#######################################################################################################
+// development version
+// by: V0JT4
+
+
+#define PLUGIN_177
+#define PLUGIN_ID_177         177
+#define PLUGIN_NAME_177       "Gases - CO2 CM1107"
+#define PLUGIN_VALUENAME1_177 "CO2"
+
+boolean Plugin_177_init = false;
+
+
+#define CM1107_ADDR   0x31 // default address
+
+#define  CM1107_CMD_CO2     0x01
+#define  CM1107_CMD_ABC     0x10
+#define  CM1107_CMD_CAL     0x03
+#define  CM1107_CMD_SN      0x1F
+#define  CM1107_CMD_VER     0x1E
+
+#define  CM1107_STATUS_INIT 0x01 //0 1 swapped compared to datasheet
+#define  CM1107_STATUS_OK   0x00
+#define  CM1107_STATUS_ERR  0x02
+#define  CM1107_STATUS_OUT  0x03
+#define  CM1107_STATUS_NCAL 0x05
+
+#define  CM1107_CAL_ON 0x00
+#define  CM1107_CAL_OFF 0x02
+
+
+
+boolean plugin_177_begin()
+{
+  Wire.begin();
```

Comment:
Do not let a plugin call `Wire.begin()`
It is already being dealt with in ESPeasy and this may disrupt other I2C communications of other plugins

Reply:
Removed